engine/server.go

```diff
@@ -0,0 +1,74 @@
+package engine
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"time"
+)
+
+type LogLevel int
+
+const (
```

Comment:
I hope we'll release without `log` prefix

Reply:
Log prefix is like a namespace to say these variables are sharing some common things. With code completion, it's easy to get all log levels when typing with `.Log`. So I hope to keep it.